dist/assests/css/header.css

```diff
@@ -1,15 +1,190 @@
 /* Main Section, Header, Nav */
-.main-header .main-header__nav {
+
```

Comment:
this file is not used, please remove it.

Reply:
updated